north/src/runtime/console.rs

```diff
@@ -201,9 +229,25 @@ async fn connection(stream: TcpStream, mut tx_main: EventTx) {
 
         // Deserialize message
         let message: Message = serde_json::from_slice(&buffer)?;
+        let tmp_installation_dir = tempdir()?;
```

Comment:
This needs a `context` because the error msg is cryptic. eg. `permission denied`